README.md

```diff
@@ -3,9 +3,9 @@
 ```diff
 - WARNING! -
 
-The ROM this repository builds cannot be 'shifted', primarily due to some hardcoded pointers which have yet
-to be dumped. Thus this repository is currently in an experimental and research phase and cannot yet be used
-traditionally as a source code base for general changes.
+This repository is a work in progress, and while it can be used to make certain changes, it's still
```

Comment:
I'm not entirely sure if changing this is the best idea right now, considering how many things are not actually shiftable, like any struct still being used in assembly, hardcoded pointers in objects that haven't been decompiled, etc.